flags.go

```diff
@@ -147,7 +147,12 @@ func (f *flagGroup) parse(context *ParseContext) (*FlagClause, error) {
 					context.Elements = context.Elements[:len(context.Elements)-pos]
 					for x := len(current) - pos - 1; x > 0; x-- {
 						// We skip all remaining elements of the group
-						context.Next()
+						purgedToken := context.Next()
+						// That's not suppose to be possible, but let's add it either way
+						if purgedToken.Type == TokenLong {
```

Comment:
I don't think that the comment matches the code, it talks about adding "it" either way but we error out here.

Reply:
I think he means: 

> This error isn't supposed to be possible, but let's handle it anyway.

Reply:
What @jonapich said 🚀